build-infer.sh

```diff
@@ -31,10 +31,12 @@ function usage() {
   echo "   java     build Java analyzer"
   echo
   echo " options:"
-  echo "   -h,--help             show this message"
   echo "   --no-opam-lock        do not use the opam.locked file and let opam resolve dependencies"
   echo "   --only-setup-opam     initialize opam, install the opam dependencies of infer, and exit"
   echo "   --user-opam-switch    use the current opam switch to install infer (default: $INFER_OPAM_DEFAULT_SWITCH)"
+  echo "   -h,--help             show this message"
+  echo "   -n,--ninja            use ninja for building clang (clang only)"
+  echo "   -s,--sequential-link  only use one process for linking (ninja only, clang only)"
```

Comment:
I think we could just have `-n,--ninja` and have it automatically enable sequential linking. This seems the only sane way to build clang to me so I see no good use case for having `-s` too.

Please keep the list alphabetically sorted (except it ignores "no-" prefixes).

Reply:
No longer relevant.